extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -339,27 +335,26 @@ export class UserNotificationManager {
    * Retrieve a promise associated to an `amp-user-notification` component
    * that is resolved when user agrees to the terms.
    * @param {string} id
-   * @return {!Promise}
+   * @return {!Promise<!AmpUserNotification>}
    */
   get(id) {
     this.managerReadyPromise_.then(() => {
       if (this.win_.document.getElementById(id) == null) {
         user.warn(TAG, `Did not find amp-user-notification element ${id}.`);
       }
     });
-    return this.getElementDeferById_(id).promise;
```

Comment:
Wondering if we had a bug here. Should this be returned inside the managerReadyPromise chain? @erwinmombay